src/mod/endpoints/mod_sofia/mod_sofia.h

```diff
@@ -405,6 +405,9 @@ struct mod_sofia_globals {
 	char guess_mask_str[16];
 	int debug_presence;
 	int debug_sla;
+	int limit_uuid_failure_code; // SIP RC For when insufficient cpu-idle
```

Comment:
the `uuid` for me is confusing on this var name, I would suggest to name it as cpu_limit_failure_code